src/spec-lang/tc/semcheck.ts

```diff
@@ -114,7 +114,7 @@ export function scAnnotation(
 ): void {
     if (node instanceof SProperty) {
         // #limit and #hint are implicitly evaluated in the old-state
```

Comment:
Comment still mentioning older `limit` and `hint` keywords. Not a blocker.

Reply:
Fixed in a1b9613